src/oss/confluent.sh

```diff
@@ -1308,6 +1310,60 @@ extract_json_config() {
     _retval="${parsed_json}"
 }
 
+produce_command() {
+    local topicname="${1}"
+    local args="$@"
+
+    if [[ "x${topicname}" == "x" ]]; then
+        echo "Missing required topic name argument in '${command_name} produce <topicname>'"
+        produce_usage
+    fi
+
+    if [[ "$args" =~ "broker-list" ]]; then
+      BROKERLIST=""
+    else
+      BROKERLIST="--broker-list localhost:9092"
+    fi
+    removestr="--value-format avro"
+    if [[ "$args" =~ "$removestr" ]]; then
+      args=${args//$removestr/}
+      AVRO=1
+    fi
+    
+    if [[ $AVRO == 1 ]]; then
```

Comment:
you could combine this into the if-statement above

Reply:
Ah yes, and this removes the need for the `avro` variable completely